### ISSUE TITLE
Assign Global Permissions to ADFS Groups

### ISSUE THREAD

**Issue by ghost** [rancher/rancher#15929]
**Rancher versions:**
rancher/rancher: **v2.0.8**

There is no way that I could find to assign ADFS groups global permissions. I know that there is a configuration option that allows you assign default global permissions for new users that log in but it's not group based. For instance all users in an ADFS group will be assigned default Admin and other groups will be assigned default user. Am I missing something and is there a way to accomplish this? 

**TLDR;** is there a way to assign ADFS groups global permissions, if not can we implement it?

**Comment by loganhz** [comment]
Close in favor of https://github.com/rancher/rancher/issues/22707
